src/etcd/client.py

```diff
@@ -784,6 +784,7 @@ def api_execute(self, path, method, params=None, timeout=None):
                               and cluster_id is not None and
                               cluster_id != self.expected_cluster_id)
                 # Update the ID so we only raise the exception once.
+                old_expected_cluster_id = self.expected_cluster_id
```

Comment:
Do you mean the assignment of `old_expected_cluster_id`, `self.expected_cluster_id`, or both?